tests/test_spark_operator.py

```diff
@@ -2,7 +2,7 @@
 To run these tests you need to have at least configured airflow in
 local mode and run:
 
-`airflow initdb`
+`airflow db init`
```

Comment:
Was it just always wrong, or they changed it in some version?

Reply:
Updated in current airflow version.